api/MediaQueryList.json

```diff
@@ -190,7 +192,7 @@
           "status": {
             "experimental": false,
             "standard_track": true,
-            "deprecated": false
+            "deprecated": true
```

Comment:
Was this actually deprecated? This is unexpected—it'd be nice to have a link in this PR to some substantiation of this.